packages/bpk-component-split-input/src/BpkSplitInput.module.scss

```diff
@@ -0,0 +1,36 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /* stylelint-disable */
+
+@import '~bpk-mixins/index';
+
+.BpkSplitInput {
+  display: flex;
+  width: 100%;
+  padding-top: $bpk-spacing-xs();
+  flex-direction: row;
+  align-items: center;
+  font-size: $bpk-font-size-base;
```

Comment:
we might want to use directly the `body-default` mixin which includes the correct line height as well?